src/sql/azdata.d.ts

```diff
@@ -3182,6 +3189,13 @@ declare module 'azdata' {
 		onDidClick: vscode.Event<any>;
 	}
 
+	export interface ImageComponent extends Component, ImageComponentProperties {
+		src: string;
```

Comment:
since you're extending `ImageComponentProperties` do you still need to duplicate all the definitions?  Shouldn't they be inherited already.

Reply:
Good catch - I went ahead and fixed up a bunch of other interface definitions that had the same issue